packages/fxa-settings/src/pages/Signin/interfaces.ts

```diff
@@ -0,0 +1,100 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import VerificationMethods from '../../constants/verification-methods';
+import VerificationReasons from '../../constants/verification-reasons';
+import { AccountAvatar } from '../../lib/interfaces';
+import { MozServices } from '../../lib/types';
+import { Integration } from '../../models';
+
+export interface AvatarResponse {
+  account: {
+    avatar: AccountAvatar;
+  };
+}
+
+export type SigninIntegration = Pick<Integration, 'type' | 'isSync'>;
+
+export type SigninContainerIntegration = Pick<
+  Integration,
+  'type' | 'isSync' | 'getService'
+>;
+
+export interface LocationState {
+  email?: string;
+  hasLinkedAccount?: boolean;
+  hasPassword?: boolean;
+}
+
+export interface SigninProps {
+  integration: SigninIntegration;
+  email: string;
+  beginSigninHandler: BeginSigninHandler;
+  cachedSigninHandler: CachedSigninHandler;
+  sessionToken?: hexstring;
+  hasLinkedAccount: boolean;
+  hasPassword: boolean;
+  serviceName: MozServices;
+  avatarData: AvatarResponse | undefined;
+  avatarLoading: boolean;
+}
+
+export interface BeginSigninOptions {
```

Comment:
I'm not seeing any references to this.

Reply:
Good catch, was probably an artifact of an early on type I didn't end up using, will remove.